next.config.js

```diff
@@ -15,6 +15,7 @@ const nextConfig = {
           params: {
             overrides: {
               removeViewBox: false,
+              cleanupIds: false,
```

Comment:
helps scoping within SVGs. without this an id of `example-xyz` will be replaced with `a`